### PR TITLE
fix(mcdu): options page text overlap and typo

### PR DESCRIPTION
**Pull request by MisterChocker**
## Summary of Changes

- Changed typo from ALT to H (altitude to height)

## Screenshots (if necessary)

image

Discord username (if different from GitHub): DerL30N#3751

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

**Review by Benjozork — APPROVED** [review]
LGTM

**Comment by marcman86** [comment]
Category: QA Tester
Name: marcman86#4907
Date of testing: 6.3.2021
Version of the sim: 1.13.17.0
PR Tested: #3827
Tier of testing: 1
Changes to observe: Option pages digit overlap
Testing technique: Tryout

Steps made to test the Pull Request:

CRZ level is FL210

1.) Spawned C&D at EDDP
2.) Turned on bat. and ext. power
3.) Looked at Options pages

Media:


Image1

Issues: -

Overall Rating: Good

Conclusions of the testing: Is fixed. But its hard to see the split between DEFAULT BARO and THR RED H.

Hope this helps :-)

File: A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/OPTIONS/A320_Neo_CDU_OPTIONS_FMGC.js [hunk 1]
```diff
@@ -28,11 +28,11 @@ class CDU_OPTIONS_FMGC {
 
         mcdu.setTemplate([
             ["A32NX OPTIONS FMGC"],
-            ["\xa0DEFAULT BARO", "THR RED ALT\xa0"],
+            ["\xa0DEFAULT BARO", "THR RED H\xa0"],
```
Comment (MisterChocker) [inline A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/OPTIONS/A320_Neo_CDU_OPTIONS_FMGC.js:31]:
```suggestion
            ["DEFAULT BARO", "THR RED H\xa0"],
```

**Review by MisterChocker — COMMENTED** [review]
Increase split between DEFAULT BARO and THR RED H to increase readability

**Comment by MisterChocker** [comment]
Improved readability
image

**Pull request merged**